src/ResponseBuilder.js

```diff
@@ -172,11 +175,28 @@ module.exports = Class.extend({
 
       this._addCacheHeaders();
 
-      return {
+      resp = {
          statusCode: this._status,
          headers: this._headers,
          body: _.isObject(this._body) ? JSON.stringify(this._body) : this._body,
       };
+
+      if (req.getEvent() && req.getEvent().requestContext && req.getEvent().requestContext.elb) {
+         // If you're running your API Lambda behind Application Load Balancer, the
```

Comment:
"API Lambda"? I guess that is accurate, just seems a little odd to have "API" as a descriptor for Lambda.